xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/resetpasswordinline.vm

```diff
@@ -114,9 +114,10 @@ $services.localization.render('xe.admin.passwordReset.instructions')
             #resetPasswordBoxStart('default')
         #end
         ## Load the configuration from a seperate document.
-        #loadConfig('XWiki.RegistrationConfig')
+        #set ($registrationConfig = $NULL)
+        #_loadConfig($registrationConfig)
```

Comment:
I'm actually wondering if we really need this now we have a script service... We could call the script service directly in the `#definePasswordFields` macro.